cmd/syncthing/cli/client.go

```diff
@@ -54,7 +56,7 @@ func (c *APIClient) Endpoint() string {
 	return url
 }
 
-func (c *APIClient) Do(req *http.Request) (*http.Response, error) {
+func (c *APIClient) Request(req *http.Request) (*http.Response, error) {
```

Comment:
`Do` matches https://golang.org/pkg/net/http/#Client.Do so this rename is slightly confusing to me